citi-onboarding/CESC-client
# (feature): Who wre are section

**What I did**

Created the Who we are section and change something in the SectionComponent.
How to test
Open the terminal and go to the folder of this project

In terminal:

git checkout develop
git pull
git checkout feature/whoweare
git pull origin feature/whoweare
yarn install
yarn start
In your browser go to http://localhost:3000/

In VScode open pages - Welcome

Test the things that you think are worth testing, even the ones that are not in this description

Make sure the Who we are page is similar to the mockup
OBS: there is somethings to change to conect front with back
